functional_tests/steps/articles.py

```diff
@@ -1,26 +0,0 @@
-from datetime import timedelta
```

Comment:
noticed while fixing a merge conflict. took the liberty of merging in to `functional_tests/features/statistical_article_page.feature`